geonode/upload/migrations/0050_alter_uploadsizelimit_max_size.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.16 on 2024-12-13 07:55
+
+from django.db import migrations, models
```

Comment:
I guess this migration is not needed. The maxsize is an environment variable that can change based on the user that creates the migrations. I think that it should be removed and we should keep a static value anyway is updatable via admin interface

Reply:
@Gpetrak you added this along with the tests. Is it needed?
https://github.com/GeoNode/geonode/blob/metadata_manager/geonode/upload/migrations/0050_alter_uploadsizelimit_max_size.py